inc/lang/zh/lang.php

```diff
@@ -326,6 +327,7 @@
 $lang['i_funcna']              = 'PHP 功能 <code>%s</code> 无法使用。也许您的服务器提供商因为某些原因禁用了它。';
 $lang['i_phpver']              = '您的 PHP 版本 <code>%s</code> 低于最低要求的 <code>%s</code>。您需要升级您的 PHP 版本。';
 $lang['i_mbfuncoverload']      = '为了运行DokuWiki，您必须在php.ini中禁用mbstring.func_overload。';
+$lang['i_urandom']             = 'DokuWiki无法为Cookie创建密码安全的数字。您可能需要检查php.ini中的open_basedir设置，以获取正确的<code> / dev / urandom </ code>访问权限。';
```

Comment:
```suggestion
$lang['i_urandom']             = 'DokuWiki 无法为 Cookie 创建密码安全的数字。您可能需要检查 php.ini 中的 open_basedir 设置，以获取正确的<code>/dev/urandom</code>访问权限。';
```

No, don't leave spaces there.